06-plot-ggplot2.Rmd

```diff
@@ -333,12 +346,10 @@ asia_oceania <- gapminder[gapminder$continent == c("Asia", "Oceania"),]
 
 ggplot(data = asia_oceania, mapping = aes(x = year, y = lifeExp)) +
   geom_line() + 
-  facet_wrap( ~ country)
+  facet_wrap(vars(country))
 ```
 
-The `facet_wrap` layer took a "formula" as its argument, denoted by the tilde
-(~). This tells R to draw a panel for each unique value in the country column
-of the gapminder dataset.
+The `facet_wrap` layer took the function `vars()` as its argument. This tells R to draw a panel for each unique value in the variable country of the gapminder dataset.
```

Comment:
```suggestion
Here we supplied the `vars` function to `facet_wrap`'s `facets` argument to easily specify which variable(s) we want to create panels for. In this case, we are telling R to draw a panel for each unique value in the variable country of the gapminder dataset.
```

Reply:
Thanks for the the suggested edit, I think I am going to make a mash-up of this with the original commit.